doc/classes/AudioStreamPlayback.xml

```diff
@@ -10,6 +10,30 @@
 		<link title="Audio Generator Demo">https://godotengine.org/asset-library/asset/526</link>
 	</tutorials>
 	<methods>
+		<method name="_get_bar_progress" qualifiers="virtual const">
+			<return type="float" />
+			<description>
+				Should return how far the playback has progressed from the last musical bar (usually 4 beats, depends on `bar_beats` in the stream import settings) to the next in range [code][0.0, 1.0][/code].
```

Comment:
I normally would reference the property directly, but I'm not sure how to do it because each stream type has its own `bar_beats` property, if any at all. Either way, to be pedantic, the stream itself influences this
```suggestion
				Should return how far the playback has progressed from the last musical bar (usually 4 beats, depending on the stream's bar beats) to the next, in the range [code][0.0, 1.0][/code].
```